packages/reakit/src/Popover/__examples__/HoverPopover/index.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { Button } from "reakit/Button";
+import {
+  usePopoverState,
+  Popover,
+  PopoverDisclosure,
+  PopoverArrow,
+  PopoverInitialState,
+} from "reakit/Popover";
+import { Separator } from "reakit/Separator";
+
+function useHoverPopoverState(initialState?: PopoverInitialState) {
+  let timeout: number;
+  const popover = usePopoverState(initialState);
+  function show() {
+    if (timeout) window.clearTimeout(timeout);
+    popover.show();
+  }
+  function hide() {
+    if (timeout) window.clearTimeout(timeout);
+    timeout = window.setTimeout(() => {
+      popover.hide();
+    }, 300);
+  }
+  return {
+    ...popover,
+    onMouseEnter: show,
+    onMouseLeave: hide,
```

Comment:
I was talking more about overriding `usePopoverState`'s own `show` and `hide`, and not returning HTML props from the state hook.

```suggestion
    show,
    hideWithDelay,
```

And then pass them to `onMouseEnter` and `onMouseLeave` on the components.

Reply:
oh, ok!